src/frontend/sharedComponents/EditScreen/index.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import {ScrollView, StyleSheet, View} from 'react-native';
+import {WHITE} from '../../lib/styles';
+
+import {ErrorBottomSheet} from '../ErrorBottomSheet';
+
+import {Photo} from '../../contexts/PhotoPromiseContext/types';
+import {DescriptionField} from './DescriptionField';
+import {ActionButtons} from './ActionButtons';
+import {MediaScrollView} from '../MediaScrollView';
+import {PresetAndLocationView} from './PresetAndLocationView';
+
+type EditScreenProps = {
+  presetName: string;
+  onPressPreset?: () => void;
+  PresetIcon: React.ReactNode;
+  showAudio?: boolean;
+  error: Error | null;
+  clearError: () => void;
+  notes: string;
+  updateNotes: (newNotes: string) => void;
+  photos: (Partial<Photo> | undefined)[];
+  audioRecordings: any[];
+  location?: {
+    lat: number | undefined;
+    lon: number | undefined;
+    accuracy: number | undefined;
+  };
+  fieldIds?: string[];
+};
+
+export const EditScreen = ({
+  presetName,
+  onPressPreset,
+  PresetIcon,
+  showAudio,
+  error,
+  clearError,
+  notes,
+  updateNotes,
+  photos,
+  audioRecordings,
+  location,
+  fieldIds,
+}: EditScreenProps) => {
+  return (
+    <View style={styles.container}>
+      <ScrollView contentContainerStyle={styles.scrollViewContent}>
+        <PresetAndLocationView
+          onPressPreset={onPressPreset}
+          presetName={presetName}
+          PresetIcon={PresetIcon}
+          location={location}
+        />
+        <DescriptionField notes={notes} updateNotes={updateNotes} />
+      </ScrollView>
+      <MediaScrollView photos={photos} audioRecordings={audioRecordings} />
+      <ActionButtons fieldIds={fieldIds} showAudio={!!showAudio} />
+      <ErrorBottomSheet error={error} clearError={clearError} />
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: WHITE,
```

Comment:
is this needed?

Reply:
https://github.com/digidem/comapeo-mobile/pull/416/commits/d3b0e6a5cd848490aa0959df5967749366a92e28